examples/tests/data_silo_plugin/main.tf

```diff
@@ -0,0 +1,60 @@
+terraform {
+  required_providers {
+    transcend = {
+      version = "0.5.1"
+      source  = "transcend.com/cli/transcend"
+    }
+  }
+}
+
+provider "transcend" {
+  url = "https://api.dev.trancsend.com/"
+}
+
+variable "schedule_frequency" {
+  type = string
+  default = "3000"
+}
+
+variable "schedule_start_at" {
+  type = string
+  default = "2022-08-16T07:00:00.000Z"
```

Comment:
I would default to some time way in the future so that it doesn't fail to deploy if the test is ran after this point in time passes (if we enforce that the start time is in the future)